criblio/appscope
# (#1283) Improve signal handler AppScope backtrace

Important note:
Currently the log transport  `CFG_FILE` is supported.
QA instructions:
The best way to test it is to use `SCOPE_ERROR_SIGNAL_HANDLER=true` and start scoping the process which will by intention end with segfault. After the process is terminated the scope.log should contain the stack trace which should be useful in further investigation
